Description: Remove partial-io as it's archived upstream
Last-Update: 2022-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,5 +60,2 @@
 
-[dev-dependencies.partial-io]
-version = "0.5"
-
 [dev-dependencies.walkdir]
--- a/src/stream/write/tests.rs
+++ b/src/stream/write/tests.rs
@@ -3,4 +3,2 @@
 
-use partial_io::{PartialOp, PartialWrite};
-
 use crate::stream::decode_all;
@@ -28,2 +26,3 @@
 
+/*
 /// Test that flush after a partial write works successfully without
@@ -71,2 +70,2 @@
     z
-}
+}*/
--- a/src/stream/tests.rs
+++ b/src/stream/tests.rs
@@ -3,4 +3,2 @@
 
-use partial_io::{PartialOp, PartialWrite};
-
 use std::io;
@@ -56,3 +54,3 @@
 
-#[test]
+/*#[test]
 fn test_try_finish() {
@@ -141,3 +139,3 @@
     );
-}
+}*/
 
